src/opencmiss/zincwidgets/fieldconditions.py

```diff
@@ -0,0 +1,100 @@
+'''
+   Copyright 2016 University of Auckland
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+'''
+from math import *
+from opencmiss.zinc.field import Field
+
+def FieldIsRealValued(field):
```

Comment:
All of the functions in the module should start with lower case.  I think these functions should all also be snake case.  I don't know how others feel but for me the form of the name should be `is_field_real_valued`.